daemons/pacemakerd/pcmkd_corosync.c

```diff
@@ -85,7 +83,7 @@ cluster_disconnect_cfg(void)
 	code;						\
 	if(rc == CS_ERR_TRY_AGAIN || rc == CS_ERR_QUEUE_FULL) {  \
 	    counter++;					\
-	    crm_debug("Retrying operation after %ds", counter);	\
+	    crm_debug("Retrying corosync operation after %ds", counter);    \
```

Comment:
In the rest of the log messages I've seen in this commit, "corosync" is capitalized.

Reply:
Yeah, we've been really inconsistent about that (for "pacemaker" too). I've even seen "CoroSync" in older documentation. It looks like the code base leans a little more to the init-cap side, so I'll go with that ...